Revit_Engine/Modify/AddMap.cs

```diff
@@ -53,12 +52,12 @@ public static TypeMap AddMap(this TypeMap typeMap, string sourceName, string des
         }
 
         /***************************************************/
-
-        [Description("Links Revit parameter with BHoM parameter for given TypeMap.")]
-        [Input("typeMap", "TypeMap")]
-        [Input("sourceName", "BHoM parameter name of type")]
-        [Input("destinationNames", "Revit parameter names to be mapped")]
-        [Output("TypeMap")]
+        
+        [Description("Links Revit parameters with BHoM type property or CustomData inside existing TypeMap.")]
+        [Input("typeMap", "TypeMap to be extended.")]
+        [Input("sourceName", "BHoM type property or CustomData name.")]
+        [Input("destinationName", "Revit parameter name to be mapped.")]
+        [Output("typeMap")]
         public static TypeMap AddMap(this TypeMap typeMap, string sourceName, IEnumerable<string> destinationNames)
```

Comment:
These don't match

Reply:
Can't believe I found the only compliance error

Reply:
But regardless, would not mind some explanation on how one maps from one BHoM param name to many Revit ones